public/index.html

```diff
@@ -3,18 +3,20 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+  <script src="https://use.fontawesome.com/35d390b5a5.js"></script>
   <title>Proxy Server</title>
 </head>
 <body>
   <div id="photos-and-info"></div>
   <div id="reviewsSection"></div>
-  <div id="photo-gallery"></div>
+  <!-- <div id="photo-gallery"></div> -->
   <div id="calendar"></div>
 
-  <script src="http://localhost:3001/bundle.js"></script>
-  <script src="http://localhost:3007/bundle.js"></script>
-  <script src="http://localhost:3005/bundle.js"></script>
-  <script src="http://localhost:3003/bundle.js"></script>
+  <script src="http://18.217.113.225:3001/bundle.js"></script>
+  <script src="http://18.222.165.232:3007/bundle.js"></script>
+  <!-- <script src="http://localhost:3005/bundle.js"></script> -->
+  <script src="http://18.223.132.12:3005/bundle.js"></script>
```

Comment:
I don't think this will work yet.  Alex still has to add you to an origin whitelist.